client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -594,10 +609,6 @@ const JetpackConnectAuthorizeForm = React.createClass( {
 			this.renderForm();
 		}
 
-		if ( this.props.plansFirst && ! this.props.selectedPlan ) {
-			return this.renderPlansSelector();
-		}
-
```

Comment:
We can also nix `renderPlansSelector` method, as it is no longer used. @johnHackworth - this is correct right, we no longer will render plans first at all, right?